students/Josh_HOff/lesson03/list_lab.py

```diff
@@ -0,0 +1,54 @@
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+#series 1
+print(fruits)
+response = input('Please add a fruit: ')
+fruits.append(response)
+print(fruits)
+response = input('Please enter a number:')
+print(response + ' {}'.format(fruits[int(response) - 1]))
+fruits = ['Pomegranates'] + fruits
+print(fruits)
+fruits.insert(0,'Blueberries')
+print(fruits)
+for i in fruits:
+    if i[0] == 'P':
+        print(i)
+fruits3 = fruits[:]
+fruits4 = fruits[:]
+#series 2    
+print(fruits)
+del fruits[-1:]
+print(fruits)
+#    response = input('Please delete a fruit: ')
+#bullet 4 in series 2
+#    fruits.remove(response)
+#    print(fruits)
+#bonus for series 2
+fruits = (fruits * 2)
+x = 1
+while x == 1:
+    response = input('Please delete a fruit: ')
+    for i in fruits:
+        if i == response:
+            fruits.remove(response)
+            x = 2
```

Comment:
the `x` seems mysterious 🙂 

if you would like to exit on the remove condition then this is a better pattern:
```
while True:
    ....
    if fruit == response:
        ...
        break
```
use `break`